adal/src/main/java/com/microsoft/aad/adal/HashMapExtensions.java

```diff
@@ -95,9 +98,9 @@
      * @return Map
      * @throws JSONException
      */
-    static Map<String, String> getJsonResponse(HttpWebResponse webResponse) throws JSONException{
+    static Map<String, String> getJsonResponse(HttpWebResponse webResponse) throws JSONException {
         final Map<String, String> response = new HashMap<>();
-        if(webResponse != null && !TextUtils.isEmpty(webResponse.getBody())) {
+        if (webResponse != null && !TextUtils.isEmpty(webResponse.getBody())) {
```

Comment:
removing this will throw another exception somewhere in checkstyle or lint. i would leave it in.

---
In reply to: [73030478](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/689#discussion_r73030478) [](ancestors = 73030478)